GoogleSignIn/Sources/GIDEMMSupport.m

```diff
@@ -44,8 +44,55 @@
 // New UIDevice system name for iOS.
 static NSString *const kNewIOSSystemName = @"iOS";
 
+// The error key in the server response.
+static NSString *const kErrorKey = @"error";
+
+// Error strings in the server response.
+static NSString *const kGeneralErrorPrefix = @"emm_";
```

Comment:
These constants are repeated from GIDEMMErrorHandler -- would it be better to have the error code mapping logic in GIDEMMErrorHandler so this class is just a wrapper? This way we have cleaner separation of responsibilities. (Also related to the discussion in other other thread about showing error UI).

Reply:
Good idea; I've removed this duplicated constants from `GIDEMMSupport`.